scipp-imaging-2D-Bench.ipynb

```diff
@@ -0,0 +1,1585 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to start\n",
+    "\n",
+    "Before starting you must ensure that `scipp`, `mantid`, and `wfm_stitching` are on your `PYTHONPATH`.\n",
+    "\n",
+    "For the first two follow instructions at: https://scipp.readthedocs.io/en/latest/getting-started/installation.html.\n",
+    "The last one can be acquired from https://git.esss.dk/wedel/wfm_stitching\n",
+    "\n",
+    "Additionally, to other package dependencies are `fabio` and `astropy`: `conda install fabio astropy`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# created by Owen Arnold (optional e-mail) on 2018-09-19\n",
+    "# last modified by Peter M. Kadletz (peter.kadletz@esss.se) on 2018-11-12\n",
+    "import fabio\n",
+    "import numpy as np\n",
+    "import glob\n",
+    "import sys\n",
+    "import csv\n",
+    "import os\n",
+    "\n",
+    "import scipp as sc\n",
+    "import numpy as np\n",
+    "from scipp import Dim\n",
+    "\n",
+    "from mantid.simpleapi import *\n",
+    "from mantid.api import AlgorithmManager\n",
+    "\n",
+    "from wfm_stitching import WFMProcessor\n",
+    "\n",
+    "# defining grouping of 2D detector pixels\n",
+    "grouping_number = 12\n",
+    "nx_target = grouping_number\n",
+    "ny_target = grouping_number\n",
+    "\n",
+    "# Number of acquired pulses per measurement\n",
+    "pulse_nr_reference = 1.0 / 770956\n",
+    "pulse_nr_sample = 1.0 / 1280381\n",
+    "pulse_nr_sample_elastic = 1.0 / 2416839\n",
+    "pulse_nr_plastic = 1.0 / 2614343\n",
+    "\n",
+    "# let's get the process started:\n",
+    "experiment_dir = \"/home/dtasev/dev/scipp_imaging/RAL_Mantid_Sep2018/\"\n",
```

Comment:
No hard-coded absolute paths please.

Reply:
I'm not sure how to handle this. Absolute paths aren't good, but having the relative aren't useful either when this notebook isn't part of the `gp2` repo. The only way it can work is to be dependent on the `gp2` repository for the `config.py`. Is that acceptable?

Reply:
I don't have to be dependent on that repository, but I think the same approach could be used. The `make_config` + products could be local to this repo.